index.js

```diff
@@ -2,6 +2,7 @@
 const fs = require('fs')
 const express = require('express')
 const path = require('path')
+require('dotenv').config();
```

Comment:
nitpick - trailing `;` - if you could make the change adhere better to the lib's eslint rules that would be great